installer/charts/tssc-pipelines/templates/tektonconfig/patch_tekton_config.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: patch-tekton-config
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  backoffLimit: 30
+  template:
+    spec:
+      serviceAccountName: {{ .Release.Name }}
+      restartPolicy: OnFailure
+      containers:
+        - name: patch-tekton-config
+          image: quay.io/openshift/origin-cli:latest
+          resources:
+            ephemeral-storage: 100Mi
+            limits:
+              memory: 128Mi
+            requests:
+              memory: 100Mi
+              cpu: 0.1
+          command:
+            - /bin/bash
+            - -c
+            - |
+              if [[ "{{ .Values.subscriptions.openshiftPipelines.managed }}" != "true" ]]; then
+                  echo "Skipping TektonConfig resource as pipeline subscription is not managed"
+                  echo "Success"
+                  exit 0
+              fi
+
+              if ! oc get tektonconfig config >/dev/null; then
+                echo "Waiting for the default TektonConfig to be instanciated"
+                sleep 10
```

Comment:
Is the sleep right before `exit 1` meant to work together with `restartPolicy` to emulate a loop?